Cargo.toml

```diff
@@ -380,7 +380,7 @@ tempfile = "3.5.0"
 test-case = "2.2.2"
 thiserror = "1.0.40"
 tiny-bip39 = "0.8.2"
-tokio = "~1.14.1"
+tokio = "1.29.1"
```

Comment:
It's too risky for us to upgrade the tokio dependency like this. We'd want the lockfile to keep tokio at 1.14.1 for the purposes of the entire v1.16 code base. But how about changing to `tokio = "1"` in the `client/` crate? This should still enable the build to select 1.14.1 from the workspace lockfile, and when the next `solana-client` crate is published, the crates.io-dependency would revert back to `tokio = "1"` and be consistent with the tokio dependency in the v1.14 version of the client crate

Reply:
@mvines So, are ready to merge the PR #32879 and backport it to the v1.16?

Reply:
maybe #32879 in v1.16 only. I don't think it's necessary in master

Reply:
for master we can just upgrade tokio directly to the newest crate if that works

Reply:
@mvines I've created the PR #32943.
It solves issue #32878 as suggested.